runtime/src/accounts_db.rs

```diff
@@ -5816,8 +5816,8 @@ pub mod tests {
         expected_lamports: u64,
     ) {
         let ancestors = vec![(slot, 0)].into_iter().collect();
-        let (account, slot) = accounts.load_slow(&ancestors, &pubkey).unwrap();
-        assert_eq!((account.lamports, slot), (expected_lamports, slot));
```

Comment:
@sakridge, Is the comparison of slot against itself here solely to cause the assert to print the slot at which it was found on failure?